### ISSUE TITLE
Feature Request: constructor param to trigger cleanup upon destruction

### ISSUE THREAD

**Issue by troglodyne** [teodesian/Selenium-Remote-Driver#330]
I've had to add reaper configuration/scripts to selenium heads due to dies in Selenium::Remote::Driver tests  (or END blocks to tests themselves) that just call $obj->quit(); to cleanup any windows. I know that leaving windows up when something dies *can* be useful for diagnostic purposes, but when not "debugging" why a test would fail, you really don't need this.

As such, I would like to request that the new() constructor be given a named argument (acting as a flag) to enable automatic reaping of the window(s) we spawned during execution. Then I could just set that in my constructors instead and not have to worry about that as much.

**Comment by teodesian** [comment]
I assume we don't want it to clean up on SIGINT?  Or should that be controllable too?

**Comment by troglodyne** [comment]
Yes, I'd presume that if the user interrupted the script (SIGINT), they would want the window to stay there for examination.

**Comment by teodesian** [comment]
Ok, will aim to make this next release.
